admin/tools/docker/replication/deployment/ncsa/start.sh

```diff
@@ -0,0 +1,117 @@
+#!/bin/bash
+
+# This file is part of {{ cookiecutter.package_name }}.
+#
+# Developed for the LSST Data Management System.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+set -e
+
+# Load parameters of the setup into the corresponding environment
+# variables
+
+. $(dirname $0)/env_svc.sh
+
+# Start database services on the master node
+#
+# The general log is temporarily disabled because it results in a huge size
+# of the log file.
+#
+#   --general-log --general-log-file="${DB_DATA_DIR}/log/${DB_CONTAINER_NAME}.general.log" \
+
+if [ ! -z "${DB_SERVICE}" ]; then
+    HOST="qserv-${MASTER}"
+    echo "[${MASTER}] starting database service"
+    ssh -n $HOST docker run \
+        --detach \
+        --name "${DB_CONTAINER_NAME}" \
+        -u 1000:1000 \
+        -v /etc/passwd:/etc/passwd:ro \
+        -v "${DB_DATA_DIR}/mysql:/var/lib/mysql" \
+        -v "${DB_DATA_DIR}/log:${DB_DATA_DIR}/log" \
+        -e "MYSQL_ROOT_PASSWORD=${DB_ROOT_PASSWORD}" \
+        -p "${DB_PORT}:${DB_PORT}/tcp" \
+        "${DB_IMAGE_TAG}" \
+        --port="${DB_PORT}" \
+        --max-connections=4096 \
+        --query-cache-size=0 \
+        --log-error="${DB_DATA_DIR}/log/${DB_CONTAINER_NAME}.error.log" \
+        --slow-query-log --slow-query-log-file="${DB_DATA_DIR}/log/${DB_CONTAINER_NAME}.slow-query.log" \
+        --log-warnings=2 \
+        --pid-file="${DB_DATA_DIR}/log/${DB_CONTAINER_NAME}.pid"
+    if [ "$?" != "0" ]; then
```

Comment:
I would prefer `$? -ne 0` test, `!=` is usually used for strings.

Reply:
Agreed